aspnetcore/fundamentals/error-handling.md

```diff
@@ -61,6 +61,34 @@ public IActionResult Error()
 
 Don't decorate the error handler action method with HTTP method attributes, such as `HttpGet`. Explicit verbs prevent some requests from reaching the method. Allow anonymous access to the method so that unauthenticated users are able to receive the error view.
 
+## Access the exception
+
+If you need to access the exception in a controller or page, obtain <xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandlerFeature> from [HttpContext.Features](xref:Microsoft.AspNetCore.Http.HttpContext.Features) and read the <xref:System.Exception?displayProperty=fullName> ([IExceptionHandlerFeature.Error](xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandlerFeature.Error)):
```

Comment:
"or if you need the orriginal request path... "

Show IExceptionHandlerPathFeature rather than IExceptionHandlerFeature, it contains both the exception and path.